src/main/java/net/glowstone/util/OpenCompute.java

```diff
@@ -135,6 +155,10 @@ public static int getLocalSize(int max) {
         return Math.min(device.getMaxWorkGroupSize(), max);
```

Comment:
Are these getLocalSize methods not supposed to have javadocs in this PR?

Reply:
Added, although CheckStyle didn't demand them.